content/en/docs/reference/access-authn-authz/authentication.md

```diff
@@ -941,24 +949,40 @@ the binary `/home/jane/bin/example-client-go-exec-plugin` is executed.
     exec:
       # Path relative to the directory of the kubeconfig
       command: "./bin/example-client-go-exec-plugin"
-      apiVersion: "client.authentication.k8s.io/v1beta1"
+      apiVersion: "client.authentication.k8s.io/v1"
+      interactiveMode: IfAvailable
```

Comment:
same comment about multiple version examples in tabs, and making the default example use "Never"

Reply:
c4a4e5ec5